.github/workflows-source/release-testing.yml

```diff
@@ -156,3 +156,10 @@ jobs:
           BAZEL_COMMAND: test --config=ci --config=systest --keep_going --test_timeout=7200 --test_env=OLD_VERSION=${{ matrix.version }}
           BAZEL_TARGETS: "//rs/tests/dre:guest_os_qualification"
           GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
+
+  repro-check:
+    name: Repro check for ${{ github.sha }}
+    uses: ./.github/workflows/repro-check.yml
+    needs: [ci-main]
```

Comment:
needs `ci-main` because artifacts need to have been uploaded first, correct?

Reply:
Yes!

Reply:
Actually it would be good to document that in a comment above this dependency.